config/initializers/okcomputer.rb

```diff
@@ -47,6 +47,9 @@ def check
     expected_count = Settings.expected_worker_count
     actual_count = Resque.workers.count
     message = "#{actual_count} out of #{expected_count} expected workers are up."
+    # it is normal for > to happen in 2 cases:
```

Comment:
This is a little hard to read, can you move these comments into the block and say "this branch happens when..."

Reply:
do you mean just before the elsif?

Reply:
yes, after the `if`

Reply:
fixed per your suggestion.